### PR TITLE
(Fix) Disable date, rate and supply fields if crowdsale has started

### LINKED ISSUE #543: (Fix) Start date input is not disabled, if crowdsale is started
https://github.com/poanetwork/ico-wizard/pull/537#issuecomment-362088690

>Likewise, if a modifiable tier is active, its start date input is enabled, but you can't really modify it.

### PR DESCRIPTION
**Pull request by fernandomg**
Closes #543 & #542 

Allow only to add addresses to the tier's whitelist if the tier has started already.

### CODE REVIEW

**Comment by fvictorio** [comment]
This lets you edit the rate input when the tier has not started, but the transaction fails. Maybe the contract doesn't allow changing the rate at all. The start date, end date and supply can be modified before the tier starts.

@vbaranov Is that the expected behavior? If so, we should always have the rate input disabled.

File: src/components/manage/index.js [hunk 1]
```diff
@@ -519,7 +524,7 @@ export class Manage extends Component {
     }
 
     const tierStartAndEndTime = (tier, index) => {
-      const disabled = !this.state.ownerCurrentUser || !tier.updatable || crowdsaleHasEnded
+      const disabled = !this.state.ownerCurrentUser || !tier.updatable || this.tierHasStarted(index) || crowdsaleHasEnded
```
Comment (vbaranov) [inline src/components/manage/index.js]: End time should be editable if crowdsale is started and not finished yet (if tier is editable)

**Comment by vbaranov** [comment]
>This lets you edit the rate input when the tier has not started, but the transaction fails. Maybe the contract doesn't allow changing the rate at all. The start date, end date and supply can be modified before the tier starts.

If the tier is editable and not started yet, a transaction to change rate shouldn't fail. Do you have another behavior @fvictorio ?

**Comment by fvictorio** [comment]
Yes, the transaction fails.

**Comment by vbaranov** [comment]
@fvictorio I have successfully changed the rate for crowdsale, that is not started yet.
These are conditions to be able to change rate:
```
function updateRate(uint newOneTokenInWei) public onlyOwner {
    assert(!finalized);
    assert(isUpdatable);
    assert(now <= startsAt);
    ...
    assert(!lastTierCntrct.finalized());
    ...
  }
```
Are all of them satisfied for your try?

**Comment by vbaranov** [comment]
@fvictorio I have successfully changed the rate for crowdsale, that is not started yet.
These are conditions to be able to change rate:
```
function updateRate(uint newOneTokenInWei) public onlyOwner {
    assert(!finalized);
    assert(isUpdatable);
    assert(now <= startsAt);
    ...
    assert(!lastTierCntrct.finalized());
    ...
  }
```
Are all of them satisfied for your try?

**Comment by fvictorio** [comment]
The problem was with the `estimateGas` function and ganache. The same thing had happened before in the invest page. It's not related with this PR.

File: src/components/manage/index.js (hunk 1 above)
Comment (fernandomg) [reply src/components/manage/index.js]: Mh! I'll need #586 to be merged before fixing this, as fixes in that PR are required for this change.

Comment (fernandomg) [reply src/components/manage/index.js]: change done

**Pull request merged**